README.md

```diff
@@ -9,3 +9,29 @@ For more information about the control container, including how to use it and ho
 
 You'll need Docker 17.06.2 or later, for multi-stage build support.
 Then run `make`!
+
+## Connecting to AWS Systems Manager (SSM)
+
+Starting from v0.5.0, users have the option to pass in their own activations for SSM. This is for users that want to set up on-premises virtual machines (VMs) in their hybrid environment as managed instances.
+
+Users can add their [own activations](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-managed-instance-activation.html) by populating the control container's user-data with a base64-encoded JSON block.
```

Comment:
Same here.

Also I don't think we need the hyphen in "user-data" or "base64-encoded".  Editor @tjkirch might throw something at me... :)

Reply:
The EC2 standard in prose is "user data", with a space and no hyphen.  "base64-encoded" should be hyphenated when used as an adjective like it is here.  :)

Reply:
Up to this point I've been using "user-data" to make it more clear that this is "host-containers.user-data" and not EC2's "user data". I take it I should just refer to both as user data going forward?

Reply:
In the specific case referred to here, I'd say it's prose, since we're still in the "introduction", talking about the user data belonging to a container and not exactly describing how to set it with a setting that happens to be called `user-data`.  If you added the word "setting" afterward, then it'd be more specific, but I don't think we're to the "here's exactly how you do this" phase in the doc quite yet.

Down below, where it's saying the control container's "user-data setting" I'd go with the hyphen, plus backticks, to make it clear that now we're talking about a technical thing, a specific setting name, `user-data`.  (You also have the example TOML, which makes it clear that a hyphen is needed for that format.)